modules/serverless-ossm-setup-with-kourier.adoc

```diff
@@ -72,13 +60,28 @@ spec:
   - from:
     - namespaceSelector:
         matchLabels:
-          serving.knative.openshift.io/system-namespace: "true"
+          knative.openshift.io/part-of: "openshift-serverless"
   podSelector: {}
   policyTypes:
   - Ingress
 ...
 ----
 <1> Add the namespace that you want to integrate with {SMProductShortName}.
++
+[NOTE]
+====
+The `knative.openshift.io/part-of: "openshift-serverless"` label was added in {ServerlessProductName} 1.22.0. If you are using {ServerlessProductName} 1.21.0 or earlier, add the `knative.openshift.io/part-of` label to the `knative-serving` and `knative-serving-ingress` namespaces:
```

Comment:
Should "If you are using {ServerlessProductName} 1.21.0 or earlier" be "If you are using {ServerlessProductName} 1.21.0 or earlier"? Presumably the statement relates to all 1.21.x versions given that the label was not added until 1.22.0. I'm not familiar enough with Serverless to know if there were any z-stream versions after 1.21.0, but if there were, presumably they need to be included in this statement too.

Reply:
Well spotted, fixed it to say `1.21.1`.